src/rust/ide/src/view/project.rs

```diff
@@ -70,16 +71,20 @@ impl ProjectView {
     /// Create a new ProjectView.
     pub async fn new(logger:&Logger, controller:controller::Project)
     -> FallibleResult<Self> {
-        let path                 = Path::new(INITIAL_FILE_PATH);
-        // This touch is to ensure, that our hardcoded module exists (so we don't require
+        let root_id              = controller.root_id;
```

Comment:
Probably not. But we need to keep track of it somewhere. Or request a way to get all available root IDs to the Engine team because it seems they can be updated with `file/addRoot` and `file/removeRoot`. (It looks like both methods are still not implemented although, so I don't know about their future)